.github/workflows/image-publish-check-on-push-scheduled.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     permissions:
       contents: read
 
-  select_helm_version_generation:
+  select_helm_tag_and_version_generation:
```

Comment:
In the job "select_helm_tag_and_version_generation," I am adding a new output named "SELECT_IMAGE_TAG_GENERATION" for tag generation. The tag generation options include "commit_hash" or "ticket_from_branch.